src/serializers/stringSerializer.js

```diff
@@ -0,0 +1,10 @@
+export default {
+  deserialize: string => string,
+  serialize: val => {
+    try {
+      return val.toString()
+    } catch (e) {
+      return `${val}`
```

Comment:
I think a good response to anything the library can't handle is to return a sane default value like empty string.